desc/io/efit_io.py

```diff
@@ -0,0 +1,234 @@
+"""Utilities for reading EFIT geqdsk files."""
+
+import numpy as np
+from matplotlib import pyplot as plt
+from scipy.interpolate import CubicSpline
+
+
+def efit_to_desc(g, M, N=0, L=None, sep_dev=0):
+    """Create a DESC equilibrium from an EFIT geqdsk file.
+
+    The equilibrium will not be solved, this just creates an equilibirum
+    with the correct boundary and profiles.
+
+    Parameters
+    ----------
+    g : path-like or dict
+        Path to geqdsk file, or dict of already read parameters.
+    M : int
+        Poloidal resolution of the equilibrium. Used for fitting the boundary.
+    N : int, optional
+        Toroidal resolution of the equilibrium.
+    L : int, optional
+        Radial resolution of the equilibrium. Used for fitting profiles.
+        If None, spline profiles will be used. Otherwise they are fit
+        to even power series in rho.
+    sep_dev : float
+        Deviation from separatrix. If sep_dev == 0, then the boundary
+         is read from the file.sep_dev = 1 corresponds to the magnetic axis.
```

Comment:
`file.sep_dev` sounds like a file structure, putting a space between would make this sentence clearer